test/nightly/global.resource.test.ts

```diff
@@ -31,15 +33,27 @@ suiteSetup(async function (this: Mocha.Context): Promise<void> {
         client = createAzureClient(testAccount.getSubscriptionContext(), CosmosDBManagementClient);
 
         // Create account
-        await Promise.all([delayCreateAccount(5, /graph/), delayCreateAccount(10, /MongoDB/), delayCreateAccount(15, /SQL/)]);
+        await Promise.all([delayOpAccount(5, /graph/, createAccount), delayOpAccount(10, /MongoDB/, createAccount), delayOpAccount(15, /SQL/, createAccount)]);
     }
 });
 
 suiteTeardown(async function (this: Mocha.Context): Promise<void> {
     if (longRunningTestsEnabled) {
         this.timeout(10 * 60 * 1000);
-        await deleteResourceGroups();
-        ext.azureAccountTreeItem.dispose();
+
+        // Delete account
+        await Promise.all([delayOpAccount(5, /MongoDB/, deleteAccount), delayOpAccount(10, /SQL/, deleteAccount)]);
+        try {
+            // If two or more of the following asserts fail, only one error will be thrown as a result.
+            await doesAccountExsit(AccountApi.MongoDB);
```

Comment:
Rather than calling `doesAccountExist` on every type of AccountApi, you should be able to iterate over the keys of `AccountList`.  I don't think that the `accountType` should matter when deleting the accounts and should be able to just pass the name of the account as a string.